iosApp/iosApp/ContentView.swift

```diff
@@ -605,7 +618,7 @@ struct ContentView: View {
         .init(
             onBack: { nearbyVM.goBack() },
             onClose: { nearbyVM.popToEntrypoint() },
-            sheetBackState: selectedDetent == .almostFull || hideMaps ? .shown : .hidden
+            backButtonPresentation: selectedDetent == .almostFull || hideMaps ? .header : .floating
```

Comment:
This should probably check `nearbyVM.navigationStack.hasFloatingBackButton()` so that it agrees with the map; that’d also replace the new copies moving the back button to the header in add and edit favorites.